hpe3parclient/client.py

```diff
@@ -4208,3 +4208,56 @@ def resyncPhysicalCopy(self, volume_name):
         info = {'action': self.RESYNC_PHYSICAL_COPY}
         response = self.http.put("/volumes/%s" % (volume_name), body=info)
         return response[1]
+
+    def admitRemoteCopyLinks(self, targetName, port_address):
+        """Adding remote copy link from soure to target.
+        :param targetName - The name of target system
+        :type - string
+        :port_address - Target system port address
+        :type- string
+        """
+        cmd = ['admitrcopylink', targetName, port_address]
+        return self._run(cmd)
+
+    def dismissRemoteCopyLinks(self, targetName, port_address):
+        """Adding remote copy link from soure to target.
+        :param targetName - The name of target system
+        :type - string
+        :port_address - Target system port address
+        :type- string
+        """
+        cmd = ['dismissrcopylink', targetName, port_address]
+        return self._run(cmd)
+
+    def admitRemoteCopyTarget(self, targetName, mode, remote_copy_group_name, volumes_in_rcg=False, source_target_volume_pairs_list=None):
```

Comment:
Only the mandatory parameters must be arguments, rest can be just the optional dict